app/src/main/res/values-en/strings.xml

```diff
@@ -113,30 +113,30 @@ The info you enter is then directly added to the OpenStreetMap in your name, wit
     <string name="at_housename">house name %s</string>
     <string name="at_housenumber">house number %s</string>
     <!-- when the user left the answer form empty -->
-    <string name="no_changes">"You did not give any answer"</string>
+    <string name="no_changes">"You did not give an answer"</string>
 
     <string name="close">"close"</string>
     <string name="confirmation_discard_title">"Discard input?"</string>
     <string name="confirmation_discard_positive">"Discard"</string>
     <string name="short_no_answer_on_button">No</string>
 
     <string name="quest_leave_new_note_title">"Leave a note instead?"</string>
-    <string name="quest_leave_new_note_description">"You can leave a public note at this location which other mappers can comment on and hopefully resolve or alternatively just hide the quest for yourself only"</string>
-    <string name="quest_leave_new_note_yes">"OK"</string>
-    <string name="quest_leave_new_note_no">"No, just hide"</string>
+    <string name="quest_leave_new_note_description">"You can leave a public note for other mappers to resolve at this location, or stop seeing this quest"</string>
+    <string name="quest_leave_new_note_yes">"Leave note"</string>
+    <string name="quest_leave_new_note_no">"Hide"</string>
```

Comment:
I think this new wording [suggested and applied](https://github.com/streetcomplete/StreetComplete/pull/4030#discussion_r874710428) by @westnordost makes it harder to understand and translate. "Hide" is not mentioned in the question, but in the button below and in other strings:

* `<string name="hid_action_description">You hid it</string>`
* `<string name="pref_title_quests_restore_hidden">Restore hidden quests</string>`
* `<string name="restore_dialog_message">Restore all quests you’ve hidden?</string>`
* `<string name="restore_hidden_success">Restored %d hidden quests</string>`

So I think mentioning "hide" to keep it consistent would make it easier to understand and translate. (Actually that's exactly @westnordost's complaint from earlier: https://github.com/streetcomplete/StreetComplete/pull/4030#discussion_r873209177)

Also, in this sentence, couldn't it be understood wrongly that others are stopped from seeing this quest, too?

My suggestion:

```suggestion
    <string name="quest_leave_new_note_description">"You can leave a public note for other mappers to resolve at this location, or hide this quest for yourself only"</string>
    <string name="quest_leave_new_note_yes">"Leave note"</string>
    <string name="quest_leave_new_note_no">"Hide"</string>
```

Reply:
Ok whatever, then let's use this then...

Clarity beats swiftness in wording.